src/core/cli/initializer.ts

```diff
@@ -154,7 +155,10 @@ class Initializer {
         'What language would you like to use in browser?',
         conf.browserLanguage
       );
-
+      conf.chromeOptions = await this.promptFolders(
+        'What chrome options would you like to use in browser?',
```

Comment:
🤔 perhaps it might be better to change its name. 
I mean, it's not really the `chromeOptions` object, it's array that extends `chromeOptions.args`

So my suggestion is to change:
- the name to `chromeOptionsAdditionalArgs`
- and the question to hmm: `What additional chromeOptions.args would you like to use in a browser?`

---
I'm not sure what it gonna return if you leave the prompt empty, did you tested it?